Example/PaymentSheet Example/PaymentSheetUITest/EmbeddedUITest.swift

```diff
@@ -14,11 +14,43 @@ class EmbeddedUITests: PaymentSheetUITestCase {
         settings.mode = .payment
         settings.integrationType = .deferred_csc
         settings.uiStyle = .embedded
+        settings.formSheetAction = .continue
         loadPlayground(app, settings)
         app.buttons["Present embedded payment element"].waitForExistenceAndTap()
-        // TODO: Test card form (see PaymentSheetVerticalUITests testUpdate)
-
-        // Selecting Alipay w/ deferred PaymentIntent...
+        
+        // Entering a card w/ deferred PaymentIntent...
+        app.buttons["Card"].waitForExistenceAndTap()
+        XCTAssertTrue(app.staticTexts["Add card"].waitForExistence(timeout: 10))
+        try! fillCardData(app, postalEnabled: true)
+        app.toolbars.buttons["Done"].tap()
```

Comment:
1. waitForExistenceAndTap() is probably more reliable.
2. TIL: didn't realize there was toolbars.buttons, i've always used "app.buttons["Done"]"